methods/messages/getMessagesMessageId.md

```diff
@@ -31,6 +31,7 @@ We only store the message contents for 30 days. Any messages older than 30 days
 | deliveryState       | One of the message delivery states `waiting` `delivered` `not-delivered `                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | deliveryCode        | Numeric value of deliver code, see table for values.                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | deliveryDescription | Message delivery description for the respective delivery code                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| segmentCount | The number of segments the message was sent as. This value will always be `1` for MMS messages.
```

Comment:
A quirk here is that any messages sent before this change was made will return a segmentCount of 0, we may want to mention that.

Reply:
Added